semanticcpg/build.sbt

```diff
@@ -6,7 +6,7 @@ libraryDependencies ++= Seq(
   "org.json4s" %% "json4s-native" % "3.6.7",
   "org.scala-lang.modules" %% "scala-collection-contrib" % "0.2.1",
   "org.scalatest" %% "scalatest" % "3.0.8" % Test,
-  "io.shiftleft" %% "fuzzyc2cpg" % Versions.fuzzyc2cpg % Test
+  "io.shiftleft" %% "fuzzyc2cpg" % Versions.fuzzyc2cpg
```

Comment:
there be dragons! 
:dragon: :dragon: :dragon: 
see? ^

IMO it would be much better to move these fixtures to a separate subproject, e.g. `codepropertygraph-testing` or so. 
I'm happy to do it, just let me know.